docs/core/entity/sensor.md

```diff
@@ -28,27 +28,28 @@ If specifying a device class, your sensor entity will need to also return the co
 | battery | % | % of battery that is left.
 | carbon_dioxide | ppm | parts per million of carbon dioxide concentration
 | carbon_monoxide | ppm | parts per million of carbon monoxide concentration
-| humidity | % | % of humidity in the air.
+| current | A | Current.
+| energy | Wh,kWh | Energy.
 | gas | m³/ft³ | Volume of gas.
+| humidity | % | % of humidity in the air.
 | illuminance | lx/lm | Light level.
+| monetary | ISO 4217 | Monetary value with a currency
 | nitrogen_dioxide | µg/m³ | Concentration of Nitrogen Dioxide |
 | nitrogen_monoxide | µg/m³ | Concentration of Nitrogen Monoxide |
 | nitrous_oxide | µg/m³ | Concentration of Nitrous Oxide |
 | ozone | µg/m³ | Concentration of Ozone |
+| power | W,kW | Power.
```

Comment:
The pm sensors should come before power, right?

Reply:
I knew I should have let that coffee kick in a bit 😆

Reply:
volatile also comes before voltage, so another sort in 0a08193.